src/components/address/[address]/AddressTokenTable.tsx

```diff
@@ -5,33 +5,53 @@ import { useWhaleApiClient } from '@contexts/WhaleContext'
 import { useEffect, useState } from 'react'
 import { CgSpinner } from 'react-icons/cg'
 import { Link } from '@components/commons/link/Link'
+import { PoolPairSymbol } from '@components/commons/PoolPairSymbol'
 
 interface AddressTokenTableProps {
   address: string
 }
 
 export function AddressTokenTable (props: AddressTokenTableProps): JSX.Element {
   const api = useWhaleApiClient()
-  const [tokensData, setTokensData] = useState<AddressToken[] | undefined>(undefined)
+  const [tokensData, setTokensData] = useState<AddressToken[]>([])
+  const [next, setNext] = useState<string | undefined>(undefined)
   const [isLoading, setIsLoading] = useState<boolean>(true)
+  const [isInitialLoad, setIsInitialLoad] = useState<boolean>(true)
 
-  function getAggregation (): void {
-    api.address.listToken(props.address).then((data: AddressToken[]) => {
-      setTokensData(data)
+  function getBalances (): void {
+    setIsLoading(true)
+
+    api.address.listToken(props.address, 10, next).then((data) => {
```

Comment:
I think should increase the default to a large number, attempt to show everything if possible?

Reply:
Looks super long and pushes the Tx table down